medusa/config.py

```diff
@@ -211,10 +211,17 @@ def parse_config(args, config_file):
         # Use the ip address instead of the fqdn when DNS resolving is turned off
         config['storage']['fqdn'] = socket.gethostbyname(socket.getfqdn())
 
-    if "CQL_USERNAME" in os.environ:
-        config['cassandra']['cql_username'] = os.environ["CQL_USERNAME"]
-    if "CQL_PASSWORD" in os.environ:
-        config['cassandra']['cql_password'] = os.environ["CQL_PASSWORD"]
+    for config_property in [
+        'cql_username',
+        'cql_password',
+        'nodetool_username',
+        'nodetool_password',
+        'sstableloader_tspw',
+        'sstableloader_kspw'
+    ]:
+        config_property_upper = "MEDUSA_{}".format(config_property.upper())
+        if config_property_upper in os.environ:
+            config['cassandra'][config_property] = os.environ[config_property_upper]
```

Comment:
Would it be possible to still support `CQL_USERNAME` and `CQL_PASSWORD`, while adding a deprecation message if they're being used?

I'm not totally comfortable with such a breaking change.

Reply:
You have raised a good point. I agree; I think supporting both and adding a deprecation message is a good idea.

I will implement this as well.